src/docfx/lib/markdown/MarkdownUtility.cs

```diff
@@ -185,7 +153,40 @@ private static (string content, object file) ReadFile(string path, object relati
             return (content, file);
         }
 
-        private static List<string> ParseMonikerRange(SourceInfo<string> monikerRange)
+        private static string GetLink(SourceInfo<string> href)
```

Comment:
still keep it inside of xxxExtension? why move it out?

Reply:
To make it so only `MarkdownUtility` references `MarkdigExtensions`, currently they references each other